include/jemalloc/internal/private_symbols.txt

```diff
@@ -74,26 +74,6 @@ arena_tdata_get
 arena_tdata_get_hard
 arenas
 arenas_tdata_cleanup
```

Comment:
This diff makes it so that we no longer have any non-inline library-internal functions which need name mangling, right?